src/runtime/auto-prompt-manager-test.js

```diff
@@ -1320,15 +1320,13 @@ describes.realWin('AutoPromptManager', (env) => {
 
   [
     {
-      actionType: 'TYPE_CONTRIBUTION',
       autoPromptType: AutoPromptType.CONTRIBUTION,
     },
     {
-      actionType: 'TYPE_SUBSCRIPTION',
       autoPromptType: AutoPromptType.SUBSCRIPTION,
     },
-  ].forEach(({actionType, autoPromptType}) => {
-    it(`should not display any prompt if UI predicate is false and page is locked for autoPromptType: ${autoPromptType}`, async () => {
+  ].forEach(({autoPromptType}) => {
```

Comment:
Would this test be more specific to the change being made if lines 1351 and 1352 were also changed to true?

Reply:
Yes, good suggestion.